ui/changes/src/common.js

```diff
@@ -299,3 +299,28 @@ export function summarizeCoverage(bugSummary) {
 
   return [lines_added, lines_covered, lines_unknown];
 }
+
+export async function getComponentRegressionMap(threshold = 0.05) {
```

Comment:
Later on, we could also add an option to take into accout test failures too (we'd need to come up with some sort of weighting to consider regression components more/less important than failing test components).